pkgs/by-name/we/wezterm/package.nix

```diff
@@ -189,8 +154,9 @@ rustPlatform.buildRustPackage rec {
     license = licenses.mit;
     mainProgram = "wezterm";
     maintainers = with maintainers; [
-      SuperSandro2000
       mimame
+      SuperSandro2000
```

Comment:
Why was the ordering changed here? mimame was added IIRC to support with Darwin because I don't own any supported hardware.

Reply:
Alphabetical order (insensitive to capitalization).

It is not like the maintainers list has any specific order anyway.